example/main.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"os"
 
-	onepassword "github.com/1password/onepassword-sdk-go"
```

Comment:
The example will error out since we removed the alias for this import. I.e. (onepassword.Items or onepassword.Vaults)

Reply:
My IDE actually gave me a `redundant alias` warning, and removing it still worked fine 🤔

Reply:
Yes, I think it takes the prefix of the package name until the first `-` as the import name, so it's redundant to explicitly import it as `onepassword`.